tests/integration_tests/test_upgrade.py

```diff
@@ -69,37 +50,79 @@ def test_clean_boot_of_upgraded_package(session_cloud: IntegrationCloud):
         'image_id': session_cloud.released_image_id,
     }
 
-    image = ImageSpecification.from_os_image()
-
-    # Get the paths to write test logs
-    output_dir = Path(session_cloud.settings.LOCAL_LOG_PATH)
-    output_dir.mkdir(parents=True, exist_ok=True)
-    base_filename = 'test_upgrade_{platform}_{os}_{{stage}}_{time}.log'.format(
-        platform=session_cloud.settings.PLATFORM,
-        os=image.release,
-        time=session_start_time,
-    )
-    before_path = output_dir / base_filename.format(stage='before')
-    after_path = output_dir / base_filename.format(stage='after')
-
-    # Get the network cfg file
-    netcfg_path = '/dev/null'
-    if image.os == 'ubuntu':
-        netcfg_path = '/etc/netplan/50-cloud-init.yaml'
```

Comment:
Was this omission from output or comparisons intentional? It seems like this check could be part of ` # Ensure important things stayed the same`

Reply:
It was intentional. I think the original intent of this check was to show that networking stayed the same, even when upgrading from ENI to netplan. In a post-xenial world, everything is netplan.

I can add the check back though. Checking that networking hasn't changed at all can still be valuable.